authentication/certificate-types-descriptions.adoc

```diff
@@ -166,7 +166,7 @@ namespace, it is system managed and rotated automatically.
 
 These certificates are managed by the system and not the user.
 
-= Control plane certificates
+== Control plane certificates
```

Comment:
It's not in your scope, but these headings should have explicit anchors.

Reply:
Thanks agree, but yeah I'd rather not add them in this PR.

FYI @ahardin-rh in case this is something you want to go back and add in. Also, I noticed that the heading levels for a lot of the discrete headings in this file (Purpose, Location, etc.) should be at a level 3. I didn't touch them since it wasn't causing a build error/warning, but they should be adjusted properly at some point.